### DISCUSSION TITLE
4 wheel drive of track (Liebherr crawler crane)

### DISCUSSION THREAD

**Discussion by vanczakp** [pybricks/support#1867] [Show and tell]
Hello all,

I made my LR1300 tracks 4 wheel driven. No more jump of the drivel wheel, runs smoothly on soft carpets as well!
It requires 2 sets of crane or get the parts of the missing two drives.
The cables of motors are long enough, and can be driven through the insided of the track.
Programs of two hubs is below. Comes with one stick or two stick control (press B to change).
Two stick control acts like one stick, only the X and Y is splitted to the two sticks.
The one stick control comes with a conversion of circle stick track to square, this means in diagonal it gives x=y=100 instead of aprox 71!

https://youtube.com/shorts/a4ftjMWxyOE?feature=share

[4wheel_dirve_crane.zip](https://github.com/user-attachments/files/17268938/4wheel_dirve_crane.zip)

**Comment by tstr14** [comment]
Are plans available?

**Reply by vanczakp** [reply]
just duplicate the drive build set. You can run the cables through the track housing.
I have two sets of crane.
